### DISCUSSION TITLE
applyMoreData自9.8.0废弃了，那么替代方法是？

### DISCUSSION THREAD

**Discussion by lantianhaijiao** [klinecharts/KLineChart#593] [General]
1.applyMoreData自9.8.0废弃了，那么替代方法是什么? applyMoreData这个方法会自动滚动到最新的位置
2.如何实现滚动到快没有数据的地方往前加载数据，setLoadDataCallback这个自动加载api会一直调用，并不会根据视口的数据情况来更新

**Comment by SamChangi** [comment]
setLoadMoreDataCallback里的callback
